src/SourceGenerators/Uno.UI.SourceGenerators.Tests/Uno.UI.SourceGenerators.Tests.csproj

```diff
@@ -1,21 +1,46 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net461</TargetFramework>
-
-		<IsPackable>false</IsPackable>
+		<TargetFrameworks>net461;net6.0</TargetFrameworks>
+		<Nullable>enable</Nullable>
+		<ImplicitUsings>enable</ImplicitUsings>
 	</PropertyGroup>
 
+	<ItemGroup Label="GlobalUsings">
+		<Using Include="Microsoft.VisualStudio.TestTools.UnitTesting" />
+		<Using Include="FluentAssertions" />
+		<Using Include="FluentAssertions.Execution" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<Compile Remove="Results\**\*.*" />
+		<None Include="Results\**\*.*" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<Compile Remove="**\*.xaml.cs" />
+		<None Include="**\*.xaml.cs" />
+	</ItemGroup>
+
 	<ItemGroup>
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
 		<PackageReference Include="MSTest.TestAdapter" Version="2.2.8" />
 		<PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
 		<PackageReference Include="coverlet.collector" Version="3.1.2" />
+		<PackageReference Include="FluentAssertions" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)' == 'net461'">
 		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.6" />
 	</ItemGroup>
 
+	<ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+		<PackageReference Include="Microsoft.CodeAnalysis" Version="4.1.0" />
+		<PackageReference Include="Uno.UI" Version="4.1.9" />
```

Comment:
The disadvantage of using Uno.UI.Skia is that it must first be assembled.

I added an alias in order to exclude the implicit use of code from Uno.UI. I also disabled all generators and parsers that come from Uno.UI.
I will check if anything else can be done to limit functionality from Uno.UI

Reply:
But yes, this does not solve the problem of having diagnostics when adding new functionality not yet presented in Uno.UI.
I think the best solution would be to explicitly include the required prebuilt dlls in this project's References folder. And update them as needed when new features are released.

Reply:
Indeed, that's not appropriate (and it increases the repro's size). I agree that the disadvantage of having to build Skia libs before is a downside, but it's one of the targets that builds the fastest, so it may not be that much trouble.